### DISCUSSION TITLE
Unable to Scan on mobile

### DISCUSSION THREAD

**Discussion by jbrumwell** [WalletConnect/walletconnect-monorepo#1938] [Developer Support]
I have tested on desktop and it works, in metamask when scanning nothing happens, in trust wallet when scanning the modal is removed and says connecting then fails with an unknown error. The same behavior is present for me when using https://react-app.walletconnect.com/ demo application. 

Is anyone able to connect on https://react-app.walletconnect.com/ via mobile scanning? Would be odd two different wallets fail on my end.

**Comment by Spider-yuchen** [comment]
I had this problem, I tried each dapps demo in 'https://github.com/WalletConnect/web-examples' and neither matemask nor imToken used on the phone could connect, I needed a demo with enough normal performance flow
